### ISSUE TITLE
Hello. I would like to have a darktable to ask me about unmounting a camera if it's not free, and unmount it if I agree.

### ISSUE THREAD

**Issue by KasheyV** [darktable-org/darktable#4380]
Hello. I would like to have a darktable to ask me about unmounting a camera if it's not free, and unmount it if I agree. 

Like it is in Entangle. Freeing me from checking why darktable tethering does not work be perfect.

Regards.

**Comment by coffeeandspaceships** [comment]
I guess this depends on #5407 first?

**Comment by jenshannoschwalm** [comment]
@johnny-bit closing this as done?

**Comment by johnny-bit** [comment]
Yeah, I think so.

@KasheyV - This is somewhat fixed: darktable thanks to @jenshannoschwalm now shows in ui information that camera might be not free and instructs user to unmount it from other application first. It doesn't do "forced-retake" like entangle can since that MIGHT lead to instabilities. The solution is now part of master and will be part of 3.6 release

I'm closing this issue for now. If you have comments/suggestions feel free to comment here (and I'll re-open the issue) or open a new issue :)
